### PR TITLE
feat(ForwardRef): Support forward ref in: Row, BoxedRow, MediaCard, Boxed, Box, ButtonLink

### PR DESCRIPTION
**Pull request by pladaria**

### CODE REVIEW

File: src/test-utils/index.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import {
     openPage,
     serverHostName,
     screen,
-    getGlobalPage,
```
Comment (pladaria) [inline src/test-utils/index.tsx:5]: this export doesn't exist

types in non source files were not checked in CI

File: flow-defs/box.js.flow [hunk 2]
```diff
@@ -36,5 +36,8 @@ declare type Props = {
   dataAttributes?: DataAttributes,
   "aria-label"?: string,
 };
-declare var Box: React.ComponentType<Props>;
+declare var Box: React.Component<{
+  ...Props,
+  ...React.RefAttributes<HTMLDivElement>,
```
Comment (pladaria) [inline flow-defs/box.js.flow:41]: this type is invalid for flow, I need to update the ts-to-flow lib

File: src/__tests__/popover-test.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import {makeTheme} from './test-utils';
 
 type Props = Omit<React.ComponentProps<typeof Popover>, 'children' | 'description' | 'target'>;
 
-const TestPopover: React.FC<Props> = (props) => (
+const TestPopover: React.FC<Props> = ({children, ...props}) => (
```
Comment (pladaria) [inline src/__tests__/popover-test.tsx:9]: Unrelated. Popover doesn't accept children

File: src/test-utils/index.tsx [hunk 4]
```diff
@@ -2,7 +2,7 @@ import {
...
+    globalPage,
```
Comment (atabel) [inline src/test-utils/index.tsx:5]: hmmm... I'd say I removed the `globalPage` export from the lib

Comment (pladaria) [reply src/test-utils/index.tsx:5]: It exists in the d.ts file

Comment (pladaria) [reply src/test-utils/index.tsx:5]: ok, I had an old library version

**Review by atabel — APPROVED** [review]

**Pull request merged**